project/Dependencies.scala

```diff
@@ -22,8 +22,10 @@ object Dependencies {
   }
 
   object Compile {
+    val akkaTyped = "com.typesafe.akka" %% "akka-actor-typed" % Versions.akka
     val akkaStream = "com.typesafe.akka" %% "akka-stream" % Versions.akka
     val akkaPersistenceQuery = "com.typesafe.akka" %% "akka-persistence-query" % Versions.akka
+    val akkaCluster = "com.typesafe.akka" %% "akka-cluster-sharding-typed" % Versions.akka
```

Comment:
`akkaClusterShardingTyped`, but not used yet

Reply:
I will remove this one